test/tests/api/v2_0/nodes_tests.py

```diff
@@ -283,6 +283,7 @@ def test_node_workflows_get(self):
                 resps.append(self.__get_data())
         for resp in resps:
             assert_not_equal(0, len(resp), message='No Workflows found for Node')
+        Api().nodes_get_workflow_by_id('fooey')
```

Comment:
@yyscamper you can see on lines 274 and 288 that the POST test depends on the GET test. The testing framework being used passes data from one test to the next so the line above is being used in POST.

Also, I'll need that line anyway when I add in a correct CIT test for the fixed api. Possible test can be seen here: https://github.com/RackHD/RackHD/pull/600

I'd test all of this locally and the CIT tests fail on cataloging and don't get to these tests. I'd like a better way to move forward on this and I'm open to suggestions. In the meantime I find this to be simple and harmless.